tests/system/CMakeLists.txt

```diff
@@ -326,6 +326,66 @@ ELSE()
 ENDIF()
 
 
+
+IF (NOT QMC_CUDA)
+
+IF (NOT QMC_COMPLEX)
+#
+# Li2 molecular dimer STO tests, all electron
+#   Reproduces UNR 1993
+#
+
+# Short tests
+  LIST(APPEND LI2STOSJS_VMC_SCALARS "totenergy" "-14.947214  0.000502") # total energy
+  LIST(APPEND LI2STOSJS_VMC_SCALARS "variance"  "  0.170618  0.001035") # energy variance
+  LIST(APPEND LI2STOSJS_DMC_SCALARS "totenergy" "-14.989438  0.000643") # total energy
+  LIST(APPEND LI2STOSJS_DMC_SCALARS "variance"  "  0.151118  0.000312") # energy variance
+
+  QMC_RUN_AND_CHECK(short-li2_sto-sj_dmc
+                    "${CMAKE_SOURCE_DIR}/tests/molecules/Li2_STO_ae"
+                    Li2.STO.short
+                    Li2.STO.short.in.xml
+                    16 1
```

Comment:
These tests are only added for MPI builds - is it preferable to have 16 procs and 1 thread or 1 proc and 16 threads (the latter would at least work on non-MPI builds)?